tmt/checks/avc.py

```diff
@@ -296,13 +298,36 @@ def create_final_report(
     # `rpm -q`, because not all non-zero exit codes mean error.
     if exc is None:
         assert output is not None
+        assert isinstance(output, CommandOutput)
 
         got_ausearch = True
-        got_denials = True
 
-        failure = list(render_command_report(label='ausearch', output=output))
-        report += failure
-        failures.append('\n'.join(failure))
+        # Include all failures in the report, even those that would be ignored later.
+        report += list(render_command_report(label='ausearch', output=output))
+
+        if check.ignore_pattern and output.stdout:
+            filtered_lines: list[str] = []
+            for line in output.stdout.splitlines():
```

Comment:
Why not iterate over the output piped through `render_command_report`? Otherwise should teach explicitly where the pattern applies, i.e. our output vs the original output.